tests/resources/Nimbus-Util.robot

```diff
@@ -405,25 +405,39 @@ Get Vsphere Version
 Deploy Nimbus NFS Datastore
     [Arguments]  ${user}  ${password}  ${additional-args}=
     ${name}=  Evaluate  'NFS-' + str(random.randint(1000,9999)) + str(time.clock())  modules=random,time
-    Log To Console  \nDeploying Nimbus NFS server: ${name}
+    Log To Console  \nDeploying Nimbus NFS testbed: ${name}
     Open Connection  %{NIMBUS_GW}
     Wait Until Keyword Succeeds  2 min  30 sec  Login  ${user}  ${password}
 
-    ${out}=  Execute Command  ${NIMBUS_LOCATION} nimbus-nfsdeploy ${name} ${additional-args}
+    ${out}=  Execute Command  ${NIMBUS_LOCATION} nimbus-testbeddeploy --testbedName nfs --runName ${name} ${additional-args}
     Log  ${out}
     # Make sure the deploy actually worked
-    Should Contain  ${out}  To manage this VM use
-    # Now grab the IP address and return the name and ip for later use
-    @{out}=  Split To Lines  ${out}
-    :FOR  ${item}  IN  @{out}
-    \   ${status}  ${message}=  Run Keyword And Ignore Error  Should Contain  ${item}  IP is
-    \   Run Keyword If  '${status}' == 'PASS'  Set Suite Variable  ${line}  ${item}
-    @{gotIP}=  Split String  ${line}  ${SPACE}
-    ${ip}=  Remove String  @{gotIP}[5]  ,
+    Should Contain  ${out}  ${name}.nfs.0' is up. IP:
+    ${out}=  Split To Lines  ${out}
+    :FOR  ${line}  IN  @{out}
+    \   ${status}=  Run Keyword And Return Status  Should Contain  ${line}  ${name}.nfs.0' is up. IP:
+    \   ${ip}=  Run Keyword If  ${status}  Fetch From Right  ${line}  ${SPACE}
+    \   Run Keyword If  ${status}  Set Suite Variable  ${nfs-ip}  ${ip}
+    \   Exit For Loop If  ${status}
+
+    :FOR  ${line}  IN  @{out}
+    \   ${status}=  Run Keyword And Return Status  Should Contain  ${line}  ${name}.nfs.1' is up. IP:
+    \   ${ip}=  Run Keyword If  ${status}  Fetch From Right  ${line}  ${SPACE}
+    \   Run Keyword If  ${status}  Set Suite Variable  ${nfs-ro-ip}  ${ip}
+    \   Exit For Loop If  ${status}
+
+    :FOR  ${line}  IN  @{out}
+    \   ${status}=  Run Keyword And Return Status  Should Contain  ${line}  ${name}.esx.0' is up. IP:
+    \   ${ip}=  Run Keyword If  ${status}  Fetch From Right  ${line}  ${SPACE}
+    \   Run Keyword If  ${status}  Set Suite Variable  ${esx-ip}  ${ip}
+    \   Exit For Loop If  ${status}
```

Comment:
these loops should be replaced with the 'Get IP' logic we are using in newer test cases like this:
https://github.com/vmware/vic/blob/master/tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.robot#L33

Reply:
Yup thanks, these are pretty handy! Updated the PR.